### PR TITLE
Dropdown bug fix

### PR DESCRIPTION
**Pull request by heartjigi**
Dropdown and some style fix
1. filter font fix 'Expertise'
2. fix my pic on edit general when 4s or etc..
3. fix expertise element align
4. can't click again bug
5. disable unstable overlay
6. remove unintended margin at dropdown

### CODE REVIEW

File: src/components/SignUp/CareerInfo.js [hunk 1]
```diff
@@ -125,14 +125,17 @@ class CareerInfo extends Component {
 
     for (let i = 0; i < this.state.pressed.length; i++) {
       this.state.pressed[i] = false;
+
```
Comment (ace0625) [inline src/components/SignUp/CareerInfo.js]: delete

File: src/utils/dropdown/optionList.js [hunk 2]
```diff
@@ -36,11 +36,10 @@ class OptionList extends Component {
   }
 
   show(items, selected, positionX, positionY, width, height, onSelect) {
-    let h = (items.length > 3) ? height * 3 + height / 4 : height * (items.length + 1);
+    let h = (items.length > 3) ? height * 3 + height / 4 : height * (items.length);
```
Comment (ifndefdeadmau5) [inline src/utils/dropdown/optionList.js]: Could we change this variable name to get better understand?

**Comment by ifndefdeadmau5** [comment]
Nice fix man, I noticed these commit log got way better than previous work!
If you change first letter to upper case in commit message, it'd be perfect.
LGTM except minor comment.

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/148/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

File: src/components/SignUp/CareerInfo.js (hunk 1 above)
Comment (heartjigi) [reply src/components/SignUp/CareerInfo.js]: Solved

File: src/utils/dropdown/optionList.js (hunk 2 above)
Comment (heartjigi) [reply src/utils/dropdown/optionList.js]: solved.

**Pull request merged**